pkg/apis/monitoring/v1/types.go

```diff
@@ -969,3 +969,62 @@ const (
 	// termination.
 	ParallelPodManagement PodManagementPolicyType = "Parallel"
 )
+
+type TracingConfig struct {
+	// clientType defines the client used to export the traces. Supported values are `http`, `grpc`, `HTTP` and `GRPC`.
```

Comment:
(nit) let's document only the "blessed" values.
```suggestion
	// clientType defines the client used to export the traces. Supported values are  `HTTP` and `GRPC`.
```